selfdrive/car/subaru/carstate.py

```diff
@@ -207,7 +207,7 @@ def get_cam_can_parser(CP):
         ("Signal6", "ES_Distance", 0),
 
         ("Counter", "ES_LKAS_State", 0),
-        ("Keep_Hands_On_Wheel", "ES_LKAS_State", 0),
+        ("LKAS_Alert_Msg", "ES_LKAS_State", 0),
         ("Empty_Box", "ES_LKAS_State", 0),
```

Comment:
this doesn't exist in the new DBC

Reply:
same with `LKAS_ENABLE_3`

Reply:
same with `LKAS_ENABLE_2`

Reply:
and `LKAS_Left_Line_Green`

Reply:
and `LKAS_Right_Line_Green`

Reply:
Updated the carstate signals to match new DBC